Simplenote/Classes/OptionsViewController.swift

```diff
@@ -356,6 +356,7 @@ private extension OptionsViewController {
         SPTracker.trackEditorCopiedInternalLink()
         UIPasteboard.general.copyInternalLink(to: note)
         NoticeController.shared.present(NoticeFactory.linkCopied())
+        SPTracker.trackPresentedNotice(kind: .linkCopied, additionalProperties: ["link_type": "internal_link"])
```

Comment:
`link_type` is used in 4 spots. It'd be neat not to repeat those strings.

Suggestion: flatten it out, less code required!

```
 enum NoticeType: String {
         case internalLinkCopied = "internal_link_copied"
         case publicLinkCopied = "public_link_copied"
```